components/CustomTable/CustomTable.tsx

```diff
@@ -0,0 +1,30 @@
+"use client";
+
+import React from "react";
+import { Table } from "antd";
+
+interface User {
+  [key: string]: any;
+}
```

Comment:
This interface name is so specific. A table can contain data related to events, tasks, not just users.